### DISCUSSION TITLE
【分享】首页大清净

### DISCUSSION THREAD

**Discussion by HyperMuteki0347** [share121/inter-knot#3222] [其他]
![1681124920229b6af39b0b6db09fb8d342bcdf6d7cb71ddee51528a7164b8c098fcf3395b08e8 0](https://github.com/user-attachments/assets/f2f90b0e-9706-43b1-acc1-1d6f760f7c03)
逆天阿b天天给我推节奏视频，本人只想刷正常视频都刷不了，不得不上屏蔽词了。
方法：手机端→我的→设置→隐私权限设置→个性化内容推荐个管理→首页推荐标题屏蔽词。
顺便分享一些我自己设置的关键词，也欢迎各路大佬再推荐一点关键词。
![Screenshot_20240820_012347](https://github.com/user-attachments/assets/198a1353-dbbb-4819-a044-758a233fdf68)
![Screenshot_20240820_012353](https://github.com/user-attachments/assets/c46d8b1d-a251-4238-9bc1-0e0463f5f00c)
![Screenshot_20240820_012358](https://github.com/user-attachments/assets/b8e6d9c4-3cc3-41a6-a3fb-0b8654f23921)

**Comment by FullMetalJacket7** [comment]
希望B站网页端原版也加一个这样的功能

**Reply by HyperMuteki0347** [reply]
其实在手机端加上屏蔽词以后pc端同样会生效

**Comment by 1492830678** [comment]
b站是已经彻底不装了，你看寒兔和纳塔的音乐的视频，那个播放差距<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by hk0852** [comment]
不行的，还要直接关闭个性化推荐

**Comment by jidianzhuxing** [comment]
其实网页端可以通过油猴插件来进行屏蔽，能用得了绳网，想必大家都会用油猴吧。
Greasy fork网址：[https://greasyfork.org/zh-CN](url)
插件名称：Bilibili 按标签、标题、时长、UP主屏蔽视频
插件作者： tjxwork
<img src="https://img-static.mihoyo.com/communityweb/upload/76cc94063d9c0efb2335e8487299cd0a.png" alt="银狼 吹泡泡" width="75" >
